payloadcms/payloadcms-prod/docker-compose.yml

```
@@ -0,0 +1,44 @@
+version: "3"
+
+services:
+  payload:
```

Can we name the image to know that it's the PROD one?

```suggestion
  payload-prod:
```